.travis.yml

```diff
@@ -67,8 +69,10 @@ jobs:
 #      - brew install gcc
       - brew install openblas lapack
       - brew install open-mpi
-    before_script: null
-    after_script : null
+    before_script:
+      - cd tests
+    after_script :
+      - cd ..
```

Comment:
In order to keep things tidy and avoid repetitions, could we move these instructions to the `script` section?